.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.d.ts
 *.js
 index.umd.js
+package-lock.json
```

Comment:
Why would you remove that file from git? The dependencies are part of the library, right? Usually the package lock is checked into git.